src/søknad/barna-dine/Barnekort.tsx

```diff
@@ -0,0 +1,97 @@
+    const { søknad, settSøknad } = useSøknadContext();
+    const [åpenEndreModal, settÅpenEndreModal] = useState(false);
+
+    const formatFnr = (fnr: string) => {
+      return fnr.substring(0, 6) + ' ' + fnr.substring(6, 11);
+    };
+
+    const ikoner = [barn1, barn2, barn3];
+    const ikon = ufødt ? ufødtIkon : ikoner[Math.floor(Math.random() * ikoner.length)];
+
+    const bosted = harSammeAdresse ? intl.formatMessage({ id: 'barnekort.adresse.registrert' }) : intl.formatMessage({ id: 'barnekort.adresse.uregistrert' });
+
+    const fjernFraSøknad = (id: string) => {
+      const nyBarneListe = søknad.person.barn.filter(b => b.id !== id);
+
+      settSøknad({...søknad, person: {...søknad.person, barn: nyBarneListe}});
+    }
+
+    const endre = (id: string) => {
+      const nyBarneListe = søknad.person.barn.filter(b => b.id !== id);
+
+      settSøknad({...søknad, person: {...søknad.person, barn: nyBarneListe}});
+
+      settÅpenModal(true);
+    }
+
+  return (
+        <div className="barnekort">
+          <div className="header">
+              <img alt="barn" className="barneikon" src={ikon} />
+          </div>
+          <div className="informasjonsboks">
+            <div className="informasjonsboks-innhold">
+              <Normaltekst className="navn">{navn}</Normaltekst>
+              <div className="informasjonselement">
+                {fnr ? <><Normaltekst>{intl.formatMessage({ id: 'barnekort.fødselsnummer' })}</Normaltekst>
+                <Normaltekst>{formatFnr(fnr)}</Normaltekst></> :
+                <><Normaltekst>{ufødt ? "TERMINDATO" : "FØDSELSDATO"}</Normaltekst>
+                <Normaltekst>{fødselsdato}</Normaltekst></>}
+              </div>
+              <div className="informasjonselement">
+                <Normaltekst>{intl.formatMessage({ id: 'barnekort.alder' })}</Normaltekst>
+                <Normaltekst>{ufødt ? "Ufødt" : alder}</Normaltekst>
+              </div>
+              <div className="informasjonselement">
+                <Normaltekst>{intl.formatMessage({ id: 'barnekort.bosted' })}</Normaltekst>
+                <Normaltekst>{bosted}</Normaltekst>
+              </div>
+              {lagtTil && id ? <div className="endre-barnekort" onClick={() => endre(id)}>
+                <Normaltekst>Endre</Normaltekst>
+              </div> : null}
+              {lagtTil && id ? <div className="endre-barnekort" onClick={() => fjernFraSøknad(id)}>
+                <Normaltekst>Fjern fra søknad</Normaltekst>
+              </div> : null}
+            </div>
+            <Modal
+          isOpen={åpenEndreModal}
+          onRequestClose={() => settÅpenModal(false)}
+          closeButton={true}
+          contentLabel="Halla"
+          >
+        <div style={{padding:'2rem 2.5rem'}}>
+          <LeggTilBarn settÅpenModal={settÅpenEndreModal} />
+        </div>
+      </Modal>
+          </div>
+        </div>
+  );
+};
+
+export default injectIntl(Barnekort);
```

Comment:
Det er nå kommet en Hook for intl! Så du kan fjerne intl fra Props og her nede, for så å paste inn 

`const intl = useIntl()`

Så skal alt fungere som før ✨ 
https://github.com/formatjs/react-intl/blob/master/docs/API.md#useintl-hook